extensions/yandex-music/src/utils.ts

```diff
@@ -0,0 +1,41 @@
+import { runAppleScript } from "run-applescript";
+import { Application, getPreferenceValues, showHUD } from "@raycast/api";
+
+// May work in other browser too, but it's confirmed to work only in these
+type SupportedBrowsers = "Safari" | "Chrome" | "Yandex";
+
+interface OsaError {
+  stderr: string;
+}
+
+function runJS(browser: SupportedBrowsers | string, code: string): string {
+  if (browser === "Safari") {
+    return `tell t to do javascript "${code}"`;
+  } else {
+    return `tell t to execute javascript "${code}"`;
+  }
+}
+
+export async function runJSInYandexMusicTab(code: string) {
+  const browser = getPreferenceValues<{ browser: Application }>().browser;
+
+  try {
+    await runAppleScript(`
+            tell application "${browser.name}"
+                repeat with w in (every window)		
+                    repeat with t in (every tab whose URL contains "music.yandex.ru") of w			
```

Comment:
I'm not sure if it's possible here, but I think it would be nice if the `music.yandex.ru` is opened in a selected browser. Otherwise, I'd show another toast with a description like "The Yandex Music page hasn't been found"